src/components/GroupList/GroupList.js

```diff
@@ -3,10 +3,9 @@
 import React from 'react'
 import { List } from 'material-ui/List'
 import GroupItem from './GroupItem'
-import type { GroupType } from 'ducks/projects'
 
 export type Props = {
-  groups: Array<GroupType>,
+  groups: Array<Object>,
```

Comment:
So not all the types properties are used in UI components, as expected, but flow requires values for them to be specified. So here we are messing up types that we are fetching from GraphQL and types that we are rendering (can be fragments of state) which causes lots of flow issues.